docker/c7n-oci

```diff
@@ -0,0 +1,73 @@
+# https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/Content/Functions/Tasks/functionsfdksandlinuxdistros_faqs.htm
+# Please refer above document if python version needs to be updated.
+FROM fnproject/python:3.8-dev as build-env
```

Comment:
its a bit unclear to me what value the python-fdk has besides restricting python version to something quite old. worse the python-fdks looks unmaintained to me.

the discussion in this issue, shows some serious both nih and lack of yagni https://github.com/fnproject/fdk-python/issues/66

Reply:
I initially used the Python FDK to align with the Function CLI, but I understand your point. I will create a new image based on Oracle Linux 8/9 Slim, which will remove the dependency on the Python FDK. Please let me know if you see any issues with this approach. In the meantime, please review the other code as well.

Reply:
there isn't any reason to use oracle linux as a base image for functions. the entire point of the oci cloud functions is that they are container based and distro and application agnostic, and invoked via a standard protocol (env var, unix http socket). that this underlying truth has been hidden is a product documentation/management failure, but it doesn't change how it works or how we should target. as a project we currently use a common base across all images, which currently avoids the mess around "enterprise" linux distributions and their derivations by not using them.

Reply:
Okay, updated the docker image.